descriptors/ModuleDescriptor-template.json

```diff
@@ -120,6 +120,19 @@
             "users.collection.get"
           ]
         },
+        {
+          "methods": [
+            "DELETE"
+          ],
+          "pathPattern": "/change-manager/jobExecutions",
+          "permissionsRequired": [
+            "change-manager.jobexecutions.post"
+          ],
+          "modulePermissions": [
+            "source-storage.snapshots.post",
+            "users.collection.get"
+          ]
+        },
```

Comment:
Permission set required to make deletion described somewhere like story AC?

Reply:
@SerhiiNosko - per our discussion, will confirm with the required stakeholders and make necessary changes if required. Thanks 🙂